README.md

```diff
@@ -116,7 +114,8 @@ To run tests, run the following command:
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-Coming soon
+- Add a list all transactions page that lists all transactions in each group in a page.
+- Add destroy feature for each category to allow users to delete their categories.
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
### ACKNOWLEDGEMENTS 

- Let's not forget to give appropriate credit to [Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).

Reply:
### CONTRIBUTING 

- Hey, let's not also forget to ensure the link to the issues pages links to this current project as this current link links to a different repo.

Reply:
#### **Live version**

- Hey, according to the project requirement, you should deploy the application using Render, Netlify, or GitHub Pages.
- You should also add the link to the `Live Demo` section of your readme. let's add this missing requirement. :pray:

Reply:
Noted :heavy_check_mark: